x-pack/plugins/apm/server/routes/mobile/route.ts

```diff
@@ -266,11 +271,110 @@ const mobileTermsByFieldRoute = createApmServerRoute({
   },
 });
 
+const mobileMainStatisticsByField = createApmServerRoute({
+  endpoint: 'GET /internal/apm/mobile-services/{serviceName}/main_statistics',
+  params: t.type({
+    path: t.type({
+      serviceName: t.string,
+    }),
+    query: t.intersection([
+      kueryRt,
+      rangeRt,
+      environmentRt,
+      t.type({
+        field: t.string,
+      }),
+    ]),
+  }),
+  options: {
+    tags: ['access:apm'],
+  },
+  handler: async (resources): Promise<MobileMainStatisticsResponse> => {
+    const apmEventClient = await getApmEventClient(resources);
+    const { params } = resources;
+    const { serviceName } = params.path;
+    const { kuery, environment, start, end, field } = params.query;
+
+    return await getMobileMainStatisticsByField({
+      kuery,
+      environment,
+      start,
+      end,
+      serviceName,
+      apmEventClient,
+      field,
+    });
+  },
+});
+
+const mobileDetailedStatisticsByField = createApmServerRoute({
+  endpoint:
+    'GET /internal/apm/mobile-services/{serviceName}/detailed_statistics',
+  params: t.type({
+    path: t.type({
+      serviceName: t.string,
+    }),
+    query: t.intersection([
+      kueryRt,
+      rangeRt,
+      offsetRt,
+      environmentRt,
+      t.type({
+        field: t.string,
+        fieldNames: jsonRt.pipe(t.array(t.string)),
+      }),
+    ]),
+  }),
+  options: {
+    tags: ['access:apm'],
+  },
+  handler: async (
+    resources
+  ): Promise<{
+    currentPeriod: import('./../../../../../../node_modules/@types/lodash/ts3.1/index').Dictionary<{
+      fieldName: string;
```

Comment:
wondering, is this import is generated automatically? do we want this?

Reply:
We use the same for transactions endpoints

Reply:
Can we remove the import() statement here and move it to a static import? It's better for typescript performance. I would also recommend a concrete type rather than something that is inferred. It is a little easier to deal with.